src/metadata.rs

```diff
@@ -390,6 +393,12 @@ pub struct EventAccountsChanged {
     pub chain_id: Chain,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct EventChainChanged {
```

Comment:
file has too many lines. please move events to separated file. My screen si too small to see whole file in one look :D

`if small is good, then smaller must be better` 
~Uncle Bob

Reply:
https://github.com/quay-rs/walletconnect-client/issues/7